bulk_query_gen/influxdb/influx_iot_common.go

```diff
@@ -120,7 +120,7 @@ func (d *InfluxIot) LightLevelEightHours(qi bulkQuerygen.Query) {
 		)
 	}
 
-	humanLabel := fmt.Sprintf(`InfluxDB (%s) field keys`, d.language)
+	humanLabel := fmt.Sprintf(`InfluxDB (%s) 8 hrs Room Light Level (Raw Data)`, d.language)
```

Comment:
Forgot to update this label initially, so fixing it here - unrelated to the primary change.